kotsadm/web/src/components/apps/AppSnapshotRestore.jsx

```diff
@@ -28,7 +28,6 @@ class AppSnapshotRestore extends Component {
   componentDidUpdate(lastProps) {
     const { match } = this.props;
     if (match.params.id !== lastProps.match.params.id) {
-      this.state.fetchRestoreDetailJob.stop();
```

Comment:
why not? if it's not stopped, `.start` won't do anything and it will keep running the old instance with the old values.

Reply:
hmmm, might not be a problem since everything is read from `props`. but stopping it won't hurt.